_includes/manuals/1.9/4.3.1_smartproxy_installation.md

```diff
@@ -19,6 +19,27 @@ You can get the latest stable code from [GitHub](https://github.com/theforeman/s
 git clone git://github.com/theforeman/smart-proxy.git
 </pre>
 
+#### Windows
+In Windows, the installation steps are different. You need to run smart-proxy from the source as well as install Ruby and Ruby DevKit. Smart-Proxy supports __ruby >= 1.9.3.__
+
+1. Run [Ruby Installer](http://rubyinstaller.org/downloads/) and add the matching and DevKit to compile native extensions. Make sure to add Ruby to `%PATH%`, you can select this option in the installer
+2. Download / clone the smart-proxy repository to a convenient location (see above, _Source Code_)
+2. Open a command prompt (`cmd.exe`) and run the following commands in order:
+  3. `ruby <devKitRoot>\dk.rb init`
```

Comment:
Hm. this renders as intended using `maruku (0.6.1)`. Might it be a style sheet? Some files do not render[1] and I was unable to use `rake server`

[1] `Error reading file ~/theforeman.org/_layouts/api.html: undefined method 'value' for #<Psych::Nodes::Document:0x000000015333c8>`